config/config.go

```diff
@@ -62,6 +63,10 @@ var configDetails = []ConfigDetail{
 		Key:         "no_prompt",
 		Description: "toggle interactive prompting in the terminal",
 	},
+	{
+		Key:         "no_check_updates",
+		Description: "toggle checking for updates",
+	},
```

Comment:
This can be removed, it will be deprecated. We have the `usage` tag above in the struct